docs/production-deployment/cloud/get-started/api-keys.mdx

```diff
@@ -81,7 +81,7 @@ Here are the differences in their permissions:
 
 - Any user can create, delete, and update their *own* API key using the Cloud UI or `tcld`.
 - Only Global Administrators and Account Owners can create, delete, and update access to API keys for all types of Service Accounts.
-- Namespace Admins can create, delete, and update access to API keys for the Namespace Scoped Service Accounts they administer.
+- Namespace Admins can create, delete, and update access to API keys for the Namespace-scoped Service Accounts they administer.
```

Comment:
Updated this to reflect what we currently have in the Cloud UI ⬇️ 

<img width="398" height="242" alt="Screenshot 2025-10-06 at 3 46 13 PM" src="https://github.com/user-attachments/assets/ff9a20e6-598e-4f10-ac25-3f6f4dcbcb94" />